pkg/sanity/controller.go

```diff
@@ -1340,35 +1340,71 @@ var _ = DescribeSanity("ListSnapshots [Controller Server]", func(sc *TestContext
 	})
 
 	It("should return snapshots that match the specified snapshot id", func() {
+		// The test creates three snapshots: one that we intend to find by
+		// snapshot ID, and two unrelated ones that must not be returned by
```

Comment:
When you say "unrelated", do you mean they won't be returned when filtering by snapshot id or filtering by source volume id?  Please clarify in the comments.

Reply:
As this is the test for snapshot IDs, the "unrelatedness" relates to filtering by snapshot IDs. I'll amend the comments.